libs/agno/agno/run/response.py

```diff
@@ -251,6 +251,7 @@ class RunResponse:
     session_id: Optional[str] = None
     team_session_id: Optional[str] = None
     parent_run_id: Optional[str] = None
+    workflow_step_id: Optional[str] = None
```

Comment:
This isn't great here... Why would the run response need this?

Reply:
updated with foreign key comment as discussed